### ISSUE TITLE
dev container) For arm, odbc requires `FROM --platform=linux/amd64`, but this makes Python fail to load in VSC devcontainer

### ISSUE THREAD

**Issue by valhuber** [microsoft/vscode-remote-release#8690]
I am using a VSC devcontainer on an M1 Mac to access SQL/Server.

I created this [apilogicserver/api_logic_server_arm_x image **with odbc**](https://github.com/ApiLogicServer/ApiLogicServer-src/blob/main/docker/api_logic_server_arm_x.Dockerfile), which note includes `FROM --platform=linux/amd64`for odbc.

 - odbc operates, but 
 - fails to load Python VSC (sometimes hangs, sometimes disabled)

[This test project](https://github.com/ApiLogicServer/beta) illustrates ***failures to load the VSCode Python extension in devcontainers specifying odbc and arm***.

**Comment by valhuber** [comment]
Any update on this?  It's holding up a release I'd like to do...

**Comment by valhuber** [comment]
After much time, I have resolved this bug.  I am not sure whether it was the odbc procedure (which I changed), or something in the docker commands, but I have a multi-platform image that is VSCode compatible, with odbc support.

It was a struggle, so if you want to pass along what I found, I have [documented it here](https://github.com/ApiLogicServer/ApiLogicServer-src/blob/main/docker/api_logic_server.Dockerfile).
